<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-N3LRL61" failures="1" tests="5" name="Product Test Execution" time="9.286" errors="0" timestamp="2025-07-30T19:40:28 IST">
  <testcase classname="Com.QA.tests.Product_Test" name="Login_Page_Test" time="0.544"/>
  <testcase classname="Com.QA.tests.Product_Test" name="Home_Page_Test" time="0.053">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Com.QA.Utilities.Utilities.validate_Url(Utilities.java:41)
at Com.QA.Pages.Home_Page.Verify_Home_page_Url(Home_Page.java:33)
at Com.QA.tests.Product_Test.Home_Page_Test(Product_Test.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- Home_Page_Test -->
  <testcase classname="Com.QA.tests.Product_Test" name="Cart_Page_Test" time="0.0">
    <skipped/>
  </testcase> <!-- Cart_Page_Test -->
  <testcase classname="Com.QA.tests.Product_Test" name="Checkout_Page_Test" time="0.0">
    <skipped/>
  </testcase> <!-- Checkout_Page_Test -->
  <testcase classname="Com.QA.tests.Product_Test" name="payment_page_Test" time="0.0">
    <skipped/>
  </testcase> <!-- payment_page_Test -->
</testsuite> <!-- Product Test Execution -->
